temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -124,3 +125,25 @@ message Namespace {
     // Will not be set if the namespace has never been modified.
     google.protobuf.Timestamp last_modified_time = 11;
 }
+
+enum FailoverType {
```

Comment:
We don't do enums on the cloud apis. 
We choose to use strings instead.

Reply:
removed.